src/autotype/AutoTypeSelectDialog.cpp

```diff
@@ -74,7 +75,7 @@ AutoTypeSelectDialog::AutoTypeSelectDialog(QWidget* parent)
             m_ui->search->setFocus();
         } else {
             // Reset to original match list
-            m_ui->view->setMatchList(m_matches, true);
+            m_ui->view->setMatchList(m_matches, -1, true);
```

Comment:
Might want to use a constant for that `-1`. Though if `selectFirst` is removed, then this becomes 0 here.

Reply:
This should be fixed now with the refactor.